submissions/RomanSaiko/html-css-popup/index.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <h1>Hooli-style Popup</h1>
+            <nav>
+                <ul class="nav-list">
+                    <li class="nav-item">
+                        <a href="#">Mail <i class="fas fa-paper-plane"></i></a>
+                    </li>
+                    <li class="nav-item">
+                        <input type="checkbox" id="submenu-checkbox" class="submenu-checkbox hidden-checkbox">
+                        <label for="submenu-checkbox"><i class="fas fa-bars"></i></label>
+                        <div class="submenu">
+                            <div class="submenu-top">
```

Comment:
If you want to show user a list of some entities it better to wrap them into `<ul>` | `<ol>` and `<li>`.
It will be beneficial for a11y